Easydict/objc/ViewController/Cell/EZTableTipsCell.m

```diff
@@ -0,0 +1,266 @@
+//
+//  EZTableTipsCell.m
+//  Easydict
+//
+//  Created by Sharker on 2024/2/18.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+#import "EZTableTipsCell.h"
```

Comment:
These are the tips feature changes that we already reverted.

Reply:
ok, removed.